helpers/windows/pdh/pdh_windows.go

```diff
@@ -207,13 +234,43 @@ func PdhGetFormattedCounterValueLong(counter PdhCounterHandle) (uint32, *PdhCoun
 }
 
 // PdhGetRawCounterValue returns the raw value of a given counter.
-func PdhGetRawCounterValue(counter PdhCounterHandle) (*PdhRawCounter, error) {
+func PdhGetRawCounterValue(counter PdhCounterHandle) (PdhRawCounter, error) {
 	var value PdhRawCounter
 	if err := _PdhGetRawCounter(counter, uintptr(unsafe.Pointer(&value))); err != nil {
-		return &value, PdhErrno(err.(syscall.Errno))
+		return value, PdhErrno(err.(syscall.Errno))
 	}
 
-	return &value, nil
+	return value, nil
+}
+
+func PdhGetRawCounterArray(counter PdhCounterHandle, filterTotal bool) (RawCouterArray, error) {
+	var bufferSize, itemCount uint32
+	if err := _PdhGetRawCounterArray(counter, &bufferSize, &itemCount, nil); err != nil {
+		if PdhErrno(err.(syscall.Errno)) != PDH_MORE_DATA {
+			return nil, PdhErrno(err.(syscall.Errno))
+		}
+		buf := make([]byte, bufferSize)
+		if err := _PdhGetRawCounterArray(counter, &bufferSize, &itemCount, &buf[0]); err != nil {
+			return nil, PdhErrno(err.(syscall.Errno))
+		}
+		items := unsafe.Slice((*pdhRawCounterItem)(unsafe.Pointer(&buf[0])), itemCount)
+		ret := make([]PdhRawCounterItem, 0, len(items))
+		for _, item := range items {
+			instance := windows.UTF16PtrToString(item.SzName)
+			if filterTotal && strings.Contains(instance, "_Total") {
```

Comment:
Do we have to worry if this is a non-English Windows system?  Just wondering if we might have something besides "_Total".

Reply:
For instance name? Nope. The name remains `_Total`. 
I did a quick test on windows VM by changing language to Spanish.

For counter name? yes, the language makes a lot of change.
Hence, I've updated the `AddCounter` function in this PR. PTAL!